apps/entry/widgets/date_range_widget.py

```diff
@@ -35,9 +36,16 @@ def update_attribute(widget, data, widget_data):
 
         'export_data': {
             'data': {
-                'excel': {
+                'version': 1,
```

Comment:
The version also needs to be like **WIDGET_ID**, since we will use it to list which attributes need updating.

Also, we should add this as a column or at the root of **export_data**.